imap_processing/tests/lo/test_lo_science.py

```diff
@@ -115,6 +116,39 @@ def segmented_pkts_fake_data():
     return dataset
 
 
+@pytest.fixture()
+def fake_spin_data():
+    dataset = xr.Dataset(
+        data_vars=dict(
+            num_completed=(["epoch"], np.array([0, 0])),
+            acq_start_sec=(["epoch"], np.array([0, 0])),
+            acq_start_subsec=(["epoch"], np.array([0, 0])),
+            acq_end_sec=(["epoch"], np.array([0, 0])),
+            acq_end_subsec=(["epoch"], np.array([0, 0])),
+        ),
+        coords=dict(epoch=(["epoch"], np.array([0, 1]))),
+    )
+    spin_fields = [
+        "start_sec_spin",
+        "start_subsec_spin",
+        "esa_neg_dac_spin",
+        "esa_pos_dac_spin",
+        "valid_period_spin",
+        "valid_phase_spin",
+        "period_source_spin",
+    ]
+
+    for spin_field in spin_fields:
+        packet_fields = [f"{spin_field}_{i}" for i in range(1, 29)]
+        fake_array_val = 0
+        for packet_field in packet_fields:
+            dataset[packet_field] = xr.DataArray(
+                np.array([fake_array_val, fake_array_val + 1]), dims="epoch"
+            )
+            fake_array_val += 2
```

Comment:
I think this might make it sequential in the output and make it easier to follow.
```suggestion
                np.array([fake_array_val, fake_array_val + 28]), dims="epoch"
            )
            fake_array_val += 1
```